flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendToolsLocator.java

```diff
@@ -158,34 +165,41 @@ private Optional<CommandResult> executeCommand(boolean logErrorOnFail,
             }
             return Optional.empty();
         }
-        List<String> stdout = new ArrayList<>();
+
+        List<String> stdout;
+        List<String> stderr;
         try {
-            String stream = IOUtils.toString(process.getInputStream(),
-                    StandardCharsets.UTF_8);
-            if (!stream.isEmpty()) {
-                Stream.of(stream.split("\\R")).forEach(stdout::add);
+            Pair<String, String> outputs = streamConsumer.get();
+            stdout = new ArrayList<>(
+                    List.of(outputs.getFirst().split(System.lineSeparator())));
+            stderr = new ArrayList<>(
+                    List.of(outputs.getFirst().split(System.lineSeparator())));
```

Comment:
I think here we should look for the second pair:
```suggestion
            stderr = new ArrayList<>(
                    List.of(outputs.getSecond().split(System.lineSeparator())));
```

Checking the implementation of the consumeProcessStreams method in FrontendUtils:
```Java
    public static CompletableFuture<Pair<String, String>> consumeProcessStreams(
            Process process) {
     ...
        return CompletableFuture.allOf(stdOut, stdErr).thenApply(
                unused -> new Pair<>(stdOut.getNow(""), stdErr.getNow("")));
    }
```
It seems the second element of the Pair is the error we are looking for here.

Reply:
Good catch!